mmv1/third_party/terraform/services/apphub/data_source_apphub_application_test.go

```diff
@@ -0,0 +1,70 @@
+package apphub_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestDataSourceApphubApplication_basic(t *testing.T) {
```

Comment:
I don't think this test is ever triggered during presubmit or nightly. This because we are only running acceptance tests with `TestAcc` prefix. Probably need to rename the test to something like `TestAccDataSourceApphubApplication_basic`